pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2866,38 +2848,35 @@ var _ = Describe("Converter", func() {
 				Expect(domainSpec.OS.BootLoader).To(BeNil())
 				Expect(domainSpec.OS.NVRam).To(BeNil())
 			})
+		})
 
-			table.DescribeTable("EFI bootloader",
-				func(secureBoot *bool, createNoSBCodeROM bool, efiCode string, efiVars string) {
-					secure := "yes"
-					if secureBoot != nil && !*secureBoot {
-						secure = "no"
-					}
-
-					createEFIRoms(createNoSBCodeROM)
+		table.DescribeTable("EFI bootloader", func(secureBoot *bool, createNoSBCodeROM bool, efiCode, efiVars, secureLoader string) {
```

Comment:
Seems like the original "EFI bootloader" block got split in 2 by this PR, this one and "Bootloader" below.
If they both serve a different purpose, can we give them more explicit names?
Also this one seems to ignore the value of `createNoSBCodeROM`.

Reply:
@jean-edouard thanks for taking a look! I extracted the 'EFI configuration detection' in a separate `Describe` block for clarity and renamed, hope it's clearer now, otherwise suggestions? Even naming tests is hard :)